src/main/resources/jenkins/branch/ItemColumn/column.jelly

```diff
@@ -27,6 +27,13 @@ THE SOFTWARE.
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
   <td style="${indenter.getCss(job)}">
     <j:choose>
+      <j:when test="${it.isOrphaned(job)}">
+        <s>
+          <a href="${jobBaseUrl}${job.shortUrl}" class='model-link inside' title="${it.getTitle(job)}">
```

Comment:
am I missing something - shouldn't this have a new class or something to set the strikethough?